### PR TITLE
Add: Allow user to remove openid

### LINKED ISSUE #1468: Allow users to remove openid
Currently a user can add multiple openids from the user Account settings. However those openids cannot be removed, they should.

### PR DESCRIPTION
**Pull request by dPliakos**
Allow user to delete an openid connection. Inserts a _font awesome_ "times" (x) right to the openid.
Fixes #1468

### CODE REVIEW

File: app/views/devise/registrations/edit.html.haml [hunk 1]
```diff
@@ -15,7 +15,10 @@
           %h4
             Currently the following openIDs are associated with your account
           - @openids.each do |openid|
-            %li= "#{openid.provider}:#{openid.email}"
+            %li
+              %span "#{openid.provider}:#{openid.email}"
+              = link_to user_openid_path(openid, user_id: current_user.id), method: :delete do
```
Comment (differentreality) [inline app/views/devise/registrations/edit.html.haml]: We should probably show a confirmation message first, before proceeding to the deletion.

File: app/views/devise/registrations/edit.html.haml [hunk 2]
```diff
@@ -15,7 +15,10 @@
...
+                %span.fa.fa-times
```
Comment (differentreality) [inline app/views/devise/registrations/edit.html.haml]: I'd make this red color, and move it in front of the openid

File: app/views/devise/registrations/edit.html.haml [hunk 3]
```diff
@@ -15,7 +15,10 @@
           %h4
             Currently the following openIDs are associated with your account
           - @openids.each do |openid|
-            %li= "#{openid.provider}:#{openid.email}"
+            %li
+              %span "#{openid.provider}:#{openid.email}"
```
Comment (differentreality) [inline app/views/devise/registrations/edit.html.haml]: We don't really need the quotes on the text that we show

File: app/views/devise/registrations/edit.html.haml (hunk 1 above)
Comment (dPliakos) [inline app/views/devise/registrations/edit.html.haml]: Does OSEM use any defaults to confirm an action?

Comment (differentreality) [inline app/views/devise/registrations/edit.html.haml]: You need to **manually** add the message

File: app/views/devise/registrations/edit.html.haml (hunk 2 above)
Comment (dPliakos) [inline app/views/devise/registrations/edit.html.haml]: The list appears a bit funny with the `X` moved to the left. 
We could remove the list bullets and use only the `X` (having both list-style role and a button role)

screenshot from 2017-06-11 15-10-12_c
screenshot from 2017-06-11 15-13-16

Comment (differentreality) [inline app/views/devise/registrations/edit.html.haml]: Let's do that!

**Comment by differentreality** [comment]
Looks good @dPliakos :D

Please squash your commits for merging!

**Comment by differentreality** [comment]
Nice work :+1:

**Pull request merged**